versions/1.0.0.md

```diff
@@ -71,6 +79,16 @@ A URL representing the issueformat (allowing a different URL format to be swappe
 
 See [Substitutions]() for more details on substitutions.
 
+### userUrlFormat (`string`)
+
+A URL representing the a user's profile URL on GitHub, Gitlab, etc.
+
+```
+{{host}}/{{user}}
+```
+
+See [Substitutions]() for more details on substitutions.
```

Comment:
I need to update this link to be valid (for this and the rest of the references)... a little difficult to do without publishing to GH first – I'll get a PR up.

Reply:
https://github.com/conventional-changelog/conventional-changelog-config-spec/pull/5 – updates the rest of the links. (`See [Substitutions](#substitutions-1) for more details on substitutions.`). Seems like we should sneak that in before `1.0.0`

Reply:
```suggestion
See [Substitutions](#substitutions-1) for more details on substitutions.
```